src/azure-cli-core/azure/cli/core/cloud.py

```diff
@@ -170,16 +171,57 @@ def _get_storage_sync_endpoint(cloud_name):
         'AzureCloud': 'afs.azure.net',
         'AzureUSGovernment': 'afs.azure.us',
     }
-    return storage_sync_endpoint_mapper.get(cloud_name, 'afs.azure.net')
+    return storage_sync_endpoint_mapper.get(cloud_name, None)
```

Comment:
why remove `afs.azure.us` ?

Reply:
To align with other endpoints, when endpoint is not set in a cloud, it means the service is not ready in that cloud, we'd better use None to raise an error instead of using the value from AzureCloud.